hooks/conan-center.py

```diff
@@ -67,6 +67,12 @@ def __init__(self, output, kb_id=None):
         if self.kb_id:
             self.kb_url = kb_url(self.kb_id)
         self._error_level = int(os.getenv("CONAN_HOOK_ERROR_LEVEL", str(NOTSET)))
+        self._set_logging_level()
+
+    def _set_logging_level(self):
+        level = os.getenv("CONAN_HOOK_LOGGING_LEVEL", str(NOTSET))
+        name_level = {"ERROR": 40, "WARNING": 30, "WARN": 30, "INFO": 20, "DEBUG": 10, "NOTSET": 0}
+        self._logging_level = int(level) if level.isdigit() else name_level[level.upper()]
```

Comment:
What if level is not in the dict?
I propose to output an error and set the warning level to `NOTSET`.
```suggestion
        if level.isdigit():
            self._logging_level = int(level)
        elif level in name_level:
            self._logging_level = name_level[level.upper()]
        else:
            self._logging_level = NOTSET
            self.error("CONAN_HOOK_LOGGING_LEVEL is set to an incorrect value")
```